bootstrap-salt.sh

```diff
@@ -4461,7 +4461,11 @@ install_amazon_linux_ami_deps() {
     # Shim to figure out if we're using old (rhel) or new (aws) rpms.
     _USEAWS=$BS_FALSE
 
-    repo_rev="$(echo "${GIT_REV:-$STABLE_REV}"  | sed 's|.*\/||g')"
+    if [ "$ITYPE" = "stable" ]; then
+        repo_rev="$(echo "${STABLE_REV}"  | sed 's|.*\/||g')"
+    else
+        repo_rev="latest"
+    fi
```

Comment:
I think you are right that repo_rev isn't actually required but I was trying to use the same variable used throughout the code. This repo_rev is also used in other parts of the bootstrap script. You can see with my latest commit I removed the if logic because we don't even really need that since as you pointed out it will be latest, and we can keep the original sed command to remove archive if its present. Are you okay with that approach?